src/infrastructure/databases.md

```diff
@@ -248,7 +248,7 @@ You can add an entry for every data domain replacing the *<domain>* placeholder
     ],
 ```
 
-The fallback database (which is also the standard database) is already configured within the Aimeos TYPO3 extension and is the one of your TYPO3 installation. If you want to move the shop related tables to their own database, add a configuration entry for **db** to your configuration file:
+The fallback database (which is also the standard database) is already configured within the Aimeos TYPO3 extension; depending on the framework you use, it is usually the database of your TYPO3/Laravel/Symfony installation. If you want to move the shop related tables to their own database, add a configuration entry for **db** to your configuration file:
```

Comment:
Would suggest:
"The fallback database (which is also the standard database) is already configured in your installation and if you want to move the shop related tables to their own database, add a configuration entry for **db** to your configuration file: